other/docs/author.md

```diff
@@ -164,43 +164,53 @@ This is a string used as import declaration's source - i.e. `'./my.macro'`.
 
 #### config (EXPERIMENTAL!)
 
-There is an experimental feature that allows users to configure your macro. We
-use [`cosmiconfig`][cosmiconfig] to read a `babel-plugin-macros` configuration which
+There is an experimental feature that allows users to configure your macro.
+
+To specify that your plugin is configurable, you pass a `configName` to `createMacro`.
+
+A configuration is created from data combined from two sources:
+We use [`cosmiconfig`][cosmiconfig] to read a `babel-plugin-macros` configuration which
 can be located in any of the following files up the directories from the
 importing file:
 
-* `.babel-plugin-macrosrc`
-* `.babel-plugin-macrosrc.json`
-* `.babel-plugin-macrosrc.yaml`
-* `.babel-plugin-macrosrc.yml`
-* `.babel-plugin-macrosrc.js`
-* `babel-plugin-macros.config.js`
-* `babelMacros` in `package.json`
+- `.babel-plugin-macrosrc`
+- `.babel-plugin-macrosrc.json`
+- `.babel-plugin-macrosrc.yaml`
+- `.babel-plugin-macrosrc.yml`
+- `.babel-plugin-macrosrc.js`
+- `babel-plugin-macros.config.js`
+- `babelMacros` in `package.json`
 
-To specify that your plugin is configurable, you pass a `configName` to
-`createMacro`:
+The content of the config will be merged with the content of the babel macros plugin
+options. Plugin options take priority.
+
+All together specifying and using the config might look like this:
 
 ```javascript
-const {createMacro} = require('babel-plugin-macros')
-const configName = 'taggedTranslations'
-module.exports = createMacro(taggedTranslationsMacro, {configName})
-function taggedTranslationsMacro({references, state, babel, config}) {
-  // config would be taggedTranslations portion of the config as loaded from `cosmiconfig`
+// .babel-plugin-macros.config.js
+module.exports = {
+  taggedTranslations: { locale: 'en_US' }
 }
-```
 
-Then to configure this, users would do something like this:
-
-```javascript
-// babel-plugin-macros.config.js
+// .babel.config.js
```

Comment:
I'd prefer `babel.config.js` and `babel-plugin-macros.config.js` rather than `.babel.config.js` and `.babel-plugin-macros.config.js`